qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,27 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def _session_client_evict(self, cmd):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.rank_asok(cmd + ['ls'])
+        self.assertIsNotNone(info)
+        size = len(info)
+        info = self.fs.rank_asok(cmd + ['evict', f'id={mount_a_client_id}'])
+        self.assertEqual(len(info), size)
+        info = self.fs.rank_asok(cmd + ['evict', f'id={mount_a_client_id}',
+                                        '--yes_i_really_mean_it'])
+        self.assertNotEqual(len(info), size)
+        self.assertNotIn(mount_a_client_id, [val['id'] for val in info])
+        info = self.fs.rank_asok(cmd + ['evict', '--yes_i_really_mean_it'])
```

Comment:
The test appears to not cover the most important thing: that eviction of all client sessions without `--yes-i-really-mean-it` fails.